app/api/user/route.ts

```diff
@@ -40,3 +41,16 @@ export async function POST(request: NextRequest) {
     return new NextResponse(`${error}`, { status: 500 })
   }
 }
+
+export async function PATCH(request: NextRequest) {
+  try {
```

Comment:
**nitpick:** This handler is image upload specific, it should be deeper - `api/user/avatar/route.ts`. Or if you want to leave it here, you'd have to set whole user update possible, not just avatar:
```ts
const updatedUser = await updateUser(email, payload)
```